src/OpenGLES/EAGLContext.cs

```diff
@@ -15,6 +20,29 @@ public EAGLContext ()
 		{
 		}
 #endif
+
+		[iOS (10,0)]
+		[TV (10,0)]
+		public bool PresentRenderBuffer (nuint target, double presentationTime)
+		{
+			return _PresentRenderbufferAtTime (target, presentationTime);
+		}
+
+		[iOS (10,3)]
+		[TV (10,2)]
+		public bool PresentRenderBuffer (nuint target, double presentationTime, PresentationMode mode)
+		{
+			switch (mode) {
+			case PresentationMode.AtTime:
+				return _PresentRenderbufferAtTime (target, presentationTime);
+				break;
+			case PresentationMode.AfterMinimumDuration:
+				return _PresentRenderbufferAfterMinimumDuration (target, presentationTime);
+				break;
+			default:
+				throw new ArgumentException ("The 'PresentationMode mode' argument needs a value.");
```

Comment:
I think something like this would be more exact and helpful:

    throw new ArgumentOutOfRangeException ($"Unknown presentation mode: {mode}", nameof (mode));